foundationdb/datadog_checks/foundationdb/check.py

```diff
@@ -217,7 +212,7 @@ def check_metrics(self, status):
                                 role_counts[rolename] = 1
 
             for role in role_counts:
-                self.gauge("foundationdb.processes_per_role." + role, role_counts[role])
+                self.gauge("foundationdb.instances", role_counts[role], ["fdb_role:" + role])
```

Comment:
Note that `foundationdb.instances` was previously a `count`, not a `gauge`, but I think that was a bug.

Reply:
Following up, I still think this is a bug, but out of scope for this pull request.